### ISSUE TITLE
Add functionality to specify a custom directory for project creation

### ISSUE THREAD

**Issue by 786raees** [Tobi-De/falco#36]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Currently, the `falco` CLI tool for Django project management does not allow users to specify a custom directory when creating a new project. This limitation requires users to manually move the project or rearrange their workspace after creation, which can be cumbersome.

**Describe the solution you'd like**
I propose adding a new option to the project creation command, `--directory` (or `-d` for short), which allows users to define a custom directory path where the new project will be initialized. This feature would enhance usability and flexibility, aligning with common practices in other project management tools.

For example:
```
falco start-project myproject --directory /path/to/custom/directory
```

**Describe alternatives you've considered**
An alternative is to manually move the project files after creation, but this approach is not as seamless and can introduce unnecessary steps for the user.

**Additional context**
Allowing users to specify the project's directory makes it easier to integrate with various development workflows and project structures. It also adds convenience for users who prefer to organize their projects in specific ways from the outset.

I believe this feature would be a valuable addition to the `falco`, and I am willing to contribute to its implementation.

Thank you for considering this enhancement.

**Comment by 786raees** [comment]
@Tobi-De What do you say?

**Comment by Tobi-De** [comment]
> @Tobi-De What do you say?

:thinking: Not sure

> Allowing users to specify the project's directory makes it easier to integrate with various development workflows and project structures

I'm curious, what are you referring to here? Do you have a real-world example? are you talking about some kind of automation process?

The `start-project` command is basically the `django-admin startproject` command with a custom template. Usually, when I want to generate a new Django project, I always move to the directory where I want that project first before running the command. Additionally, to replicate what that option would do, you could simply run 

```sh
falco start-project my_project && mv my_project /custom/path/my_project
```

So right now, I don't really see when this would be useful. Maybe you have some specific scenario in mind?

**Comment by 786raees** [comment]
Ah you are right but django provide the ability to specify dir name after project name.
and I was talking about micro service architecture above.

**Comment by Tobi-De** [comment]
> Ah you are right but django provide the ability to specify dir name after project name. and I was talking about micro service architecture above.

Oh yes, you're right, I never realized that :laughing: 
Then it is a simple addition then, it could work exactly like django, a second optional argument that default to the current directory and the value get passed to the `django-admin startproject` command

**Comment by 786raees** [comment]
Exactly I was talking about this.

**Comment by Tobi-De** [comment]
Implemented in 087deace9faf8b835514cc31269e92d7e1a9c572
